src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -132,3 +132,65 @@
     line-height: 1.3;
   }
 }
+
+.barStat {
```

Comment:
Maybe this could be renamed to something like "barsContainer"? I misinterpreted "stat" to mean "statistic", as in a single statistic as opposed to multiple statics, and therefore I thought this was just one bar and its associated label and value rather than an element that contained all of the bars.